doc/user-docs/ThirdPartyToolProfiles.md

```diff
@@ -16,6 +16,20 @@ Assuming that you've installed Anaconda into `%USERPROFILE%\Anaconda3`:
 }
 ```
 
+## AWS Shell
+
+Assuming that you've installed via pip to python 3.1 into `C:\Python310\Scripts`:
+
+```json
+{
+    "commandline": "cmd.exe /k \"C:\\Python310\\Scripts\\aws-shell.exe",
```

Comment:
This one seems sus to me - the AWS docs themselves (https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)

suggest installing the aws cli via an MSI, which I would doubt would install into the same place as the pip install path. I think if we're going to add this to the docs here, we should try to align with their official docs as best we can.